templates/users/default.dev.j2

```diff
@@ -2,13 +2,15 @@
 
 {%- set config = item.config|default([]) -%}
 
+# If you come from bash you might have to change your $PATH.
+{{ macros.config_row(config, 'export PATH', '# export PATH=$HOME/bin:/usr/local/bin:$PATH', true) }}
+
 # Path to your oh-my-zsh installation.
 {{ macros.config_row(config, 'export ZSH', manala_ohmyzsh_dir) }}
 
-# Set name of the theme to load.
-# Look in ~/.oh-my-zsh/themes/
-# Optionally, if you set this to "random", it'll load a random theme each
-# time that oh-my-zsh is loaded.
+# Set name of the theme to load. Optionally, if you set this to "random"
+# it'll load a random theme each time that oh-my-zsh is loaded.
```

Comment:
Dude, it's simply upstream :)

https://github.com/robbyrussell/oh-my-zsh/commit/9e81b79d90606bc0afb6ce17867ff4d42df3ab37